crates/fuzzing/src/oracles.rs

```diff
@@ -133,6 +133,10 @@ pub fn instantiate_with_config(
         Err(e) if e.to_string().contains("resource limit exceeded") => {}
         // Also allow errors related to fuel consumption
         Err(e) if e.to_string().contains("all fuel consumed") => {}
+        // Currently we instantiate with a `Linker` which can't instantiate
+        // every single module under the sun due to using name-based resolution
+        // rather than positional-based resolution
+        Err(e) if e.to_string().contains("incompatible import type") => {}
```

Comment:
We might want to consolidate these so that we only `to_string` once, which should help our fuzzing throughput a little bit.

Reply:
👍